systest-docker/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  f5-oslbaasv2-systest:
+    image: f5-openstack-systest-ansible-env:latest
+    container_name: f5-oslbaasv2-systest-con
+    working_dir: /f5-oslbaasv2-systest
+    build: .
+    volumes:
+      - ~/.ssh:/root/.ssh
+      - $HOMEDIR/conf.d:/f5-oslbaasv2-systest/conf.d
```

Comment:
The `$HOMEDIR` is `systest-docker` not the root directory of f5-oslbaasv2-systest.
What's more, is it possible to mount a single volume, like `- $HOMEDIR/..:/f5-oslbaasv2-systest`?